sdk/hdinsight/Azure.ResourceManager.HDInsight/CHANGELOG.md

```diff
@@ -1,14 +1,11 @@
 # Release History
 
-## 1.1.0-beta.2 (Unreleased)
+## 1.1.0-beta.2 (2023-06-16)
```

Comment:
Can we remove the beta for this version? From my side, I hope after our test we can release one version without beta

Reply:
Sure, updated.